Source/WebKit/Shared/Cocoa/SharedCARingBuffer.h

```diff
@@ -44,15 +44,17 @@ class SharedCARingBufferBase : public WebCore::CARingBuffer {
     Ref<SharedMemory> m_storage;
 };
 
+struct ConsumerSharedCARingBufferHandle {
+    SharedMemory::Handle memory;
+    size_t frameCount;
```

Comment:
Default initializing to 0 couldn't hurt, and might prevent uninitialized memory.